shell/go.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Parses the embedding project's `.tool-versions` file to find which is the
+# currently active version of Go, then invokes it explicitly through `asdf`.
+#
+# The idea here is that `some_project/.bootstrap/shell/go.sh` will will always
+# point to the "right" go for that particular `some_project`.
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+GOVERSION=$(grep golang "${DIR}/../../.tool-versions" | tail -n 1 | awk '{ print $2 }')
+
+if [ -z "${GOVERSION}" ]; then
+  echo "Unable to find Go version for this project" >&2
```

Comment:
isn't it this -
```suggestion
  echo "Unable to find Go version for this project" 1>&2
```
maybe im wrong, not the _best_ with bash

Reply:
No, `>` implies `1>`

Reply:
nice

Reply:
I see what you mean, but I think just `>&2` works.  (I guess the `1` part is implicit?)

https://stackoverflow.com/questions/31656645/how-do-i-echo-directly-on-standard-output-inside-a-shell-function